.github/workflows/codcoverage.yml

```diff
@@ -1,4 +1,4 @@
-name: graphql Code coverage
+name: Codecoverage
 
 on:
```

Comment:
Any particular reason on how we merge this file? maybe the codecoverage if not pushed together is fake?

I would like to bisect different action for different package because we can have isolate the error in the package instead to have only one failure and debug the Github CI.

However, maybe I'am missing some other good reason to have this change?